src/v/cluster/partition_probe.cc

```diff
@@ -233,7 +234,8 @@ void replicated_partition_probe::setup_public_metrics(const model::ntp& ntp) {
         sm::make_total_bytes(
           "request_bytes_total",
           [this] { return _bytes_fetched; },
-          sm::description("Total number of bytes consumed per topic"),
+          sm::description("Total number of bytes consumed per topic (not all "
```

Comment:
We should just use the internal metric description I think. I think this one is more misleading that the internal one.

Reply:
Makes sense, copied the internal description.